build/make/test-integration.mk

```
@@ -34,19 +35,13 @@ ifneq ($(strip $(PRE_INTEGRATIONTESTS)),)
 	@make $(PRE_INTEGRATIONTESTS)
 endif
 
-	@mkdir -p $(INTEGRATION_TEST_DIR)
-	@echo 'mode: set' > ${INTEGRATION_TEST_REPORT}
+	mkdir -p $(INTEGRATION_TEST_DIR)
+	echo 'mode: set' > $(INTEGRATION_TEST_REPORT)
 	@rm -f $(INTEGRATION_TEST_LOG) || true
-	@for PKG in $(PACKAGES_FOR_INTEGRATION_TEST) ; do \
-    ${GO_CALL} test -tags=${GO_BUILD_TAG_INTEGRATION_TEST} -v $$PKG -coverprofile=${INTEGRATION_TEST_REPORT}.tmp 2>&1 | tee $(INTEGRATION_TEST_LOG).tmp ; \
-		cat ${INTEGRATION_TEST_REPORT}.tmp | tail +2 >> ${INTEGRATION_TEST_REPORT} ; \
-		rm -f ${INTEGRATION_TEST_REPORT}.tmp ; \
-		cat $(INTEGRATION_TEST_LOG).tmp >> $(INTEGRATION_TEST_LOG) ; \
-		rm -f $(INTEGRATION_TEST_LOG).tmp ; \
-	done
+	@$(GO_CALL) test ./... -v -tags=${GO_BUILD_TAG_INTEGRATION_TEST} -coverpkg=./... -coverprofile=${INTEGRATION_TEST_REPORT} -run ${INTEGRATION_TEST_NAME_PATTERN} 2>&1 | tee $(INTEGRATION_TEST_LOG)
 	@cat $(INTEGRATION_TEST_LOG) | $(GO_JUNIT_REPORT) > $@
 	@if grep '^FAIL' $(INTEGRATION_TEST_LOG); then \
-		exit 1; \
+	exit 1; \
 	fi
```

Sadly the indentation was lost in transition which helps to understand this code as Bash (in contrast to Makefile) syntax

```suggestion
		exit 1; \
	fi
```